language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/PositionUtil.java

```diff
@@ -186,4 +188,20 @@ public static int getPositionOffset(Position position, SyntaxTree syntaxTree) {
         LinePosition linePos = LinePosition.from(position.getLine(), position.getCharacter());
         return syntaxTree.textDocument().textPositionFrom(linePos);
     }
+
+    /**
+     * Returns ballerina text edit for a given lsp4j text edit.
+     *
+     * @param syntaxTree syntax tree
+     * @param textEdit   lsp4j text edit
+     * @return Ballerina text edit
+     */
+    public static TextEdit getTextEdit(SyntaxTree syntaxTree, org.eclipse.lsp4j.TextEdit textEdit) {
```

Comment:
Is PositionUtil the best place for this?

Reply:
How about commonUtil?